### DISCUSSION TITLE
i18n  on url of the documents

### DISCUSSION THREAD

**Discussion by AngeloCG97** [facebook/docusaurus#5006] [General] [answered]
I want to know if is there any way to translate the url of the documents

**Comment by slorber** [comment, marked as answer]
In theory you can use the slug frontmatter on documents to customize the URL of localized docs

But we don't have first class support for that. The way the "language switch" is implemented will redirect to the same doc URL with a different locale prefix, so if 2 language have different slugs for the same doc, that language switch will fail.

We don't have a way to map `/fr/doc-en-francais` to `/en/doc-in-english`. 

It is not easy to build and apparently localizing slugs has little SEO value, so we made it out of the i18n scope: https://docusaurus.io/docs/i18n/introduction#i18n-non-goals

**Reply by AngeloCG97** [reply]
Thanks for the information
